discovery-provider/src/queries/search.py

```diff
@@ -40,6 +43,145 @@ def search_full():
 def search_autocomplete():
     return search(True)
 
+@bp.route("/search/tags", methods=("GET",))
+def search_tags():
+    logger.warning('search tags working')
+    search_str = request.args.get("query", type=str)
+    if not search_str:
+        raise exceptions.ArgumentError("Invalid value for parameter 'query'")
+
+    user_tag_count = request.args.get("user_tag_count", type=str)
+    if not user_tag_count:
+        user_tag_count = "1"
```

Comment:
@hareeshnagaraj this is defaulted to 1 here but 2 in libs, was this intentional?

Reply:
no it was not, i will switch this to 2 here - i think that makes more sense